src/mongocxx/read_preference.cpp

```diff
@@ -114,6 +114,23 @@ stdx::optional<std::chrono::seconds> read_preference::max_staleness() const {
     return std::chrono::seconds{staleness};
 }
 
+read_preference& read_preference::hedge(bsoncxx::document::view_or_value hedge) {
+    libbson::scoped_bson_t hedge_bson{hedge.view()};
```

Comment:
Since we are taking the container type `view_or_value` here, let's `std::move` it into the scoped_bson_t.  Internally scoped_bson_t is a "sink" for this object, so if we `std::move` in, it can `std::move` it all the way down.

Reply:
Done